geniza/footnotes/models.py

```diff
@@ -463,11 +471,13 @@ class Footnote(TrackChangesModel):
     TRANSLATION = "T"
     DISCUSSION = "D"
     DIGITAL_EDITION = "X"
+    DIGITAL_TRANSLATION = "Y"
     DOCUMENT_RELATION_TYPES = (
         (EDITION, _("Edition")),
         (TRANSLATION, _("Translation")),
         (DISCUSSION, _("Discussion")),
         (DIGITAL_EDITION, _("Digital Edition")),
+        (DIGITAL_TRANSLATION, _("Digital Translation")),
```

Comment:
do admin labels need to be translated? or will this term be used on the public site somewhere?